src/main/java/org/broadinstitute/dsde/consent/ontology/service/ElasticSearchAutocompleteAPI.java

```diff
@@ -53,19 +51,18 @@ public ElasticSearchAutocompleteAPI(ElasticSearchConfiguration configuration) {
     private List<TermResource> executeSearch(String query, int limit, Boolean thinFilter) {
         List<TermResource> termList = new ArrayList<>();
         try {
-            Map<String, String> params = new HashMap<>();
-            params.put("size", String.valueOf(limit));
-            Response esResponse = client.performRequest(
-                "GET",
-                ElasticSearchSupport.getSearchPath(configuration.getIndex()),
-                params,
-                new NStringEntity(query, ContentType.APPLICATION_JSON),
-                ElasticSearchSupport.jsonHeader);
-            if (esResponse.getStatusLine().getStatusCode() != 200) {
-                logger.error("Invalid search request: " + esResponse.getStatusLine().getReasonPhrase());
-                throw new InternalServerErrorException();
+            String stringResponse;
+            try {
+                stringResponse = client.
+                    target(ElasticSearchSupport.getSearchPath(configuration)).
+                    queryParam("size", String.valueOf(limit)).
+                    request(MediaType.APPLICATION_JSON).
+                    post(Entity.entity(query, MediaType.APPLICATION_JSON)).
+                    readEntity(String.class);
+            } catch (Exception e) {
+                logger.error("Invalid query request: " + e.getMessage());
+                throw new InternalServerErrorException(e.getMessage());
```

Comment:
Unfortunately, yes - the jersey client doesn't let you put a post body into a get request. Fortunately, the ES endpoint accepts both POST and GET (see about half-way down https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-body.html)